src/OrbitGl/ThreadTrack.cpp

```diff
@@ -271,20 +270,19 @@ void ThreadTrack::UpdatePositionOfSubtracks() {
   const float space_between_subtracks = layout_->GetSpaceBetweenThreadPanes();
 
   const Vec2 pos = GetPos();
-  float current_y =
-      header_->GetPos()[1] + header_->GetHeight() + layout_->GetTrackContentTopMargin();
+  float current_y = GetPos()[1] + layout_->GetTrackContentTopMargin();
 
-  thread_state_bar_->SetPos(pos[0], current_y);
+  thread_state_bar_->SetPos(pos[0] + header_->GetWidth(), current_y);
```

Comment:
Since you are always using `pos[0] + header_->GetWidth()` and to avoid repetition of code, I would prefer to include the header_width in the variable pos.

Reply:
done.